folly/Portability.h

```diff
@@ -107,11 +107,18 @@ constexpr bool kHasUnalignedAccess = false;
 #define FOLLY_PPC64 0
 #endif
 
+#if defined(__s390__)
+#define FOLLY_S390 1
```

Comment:
It's probably worth having this macro and the constexpr below refer to the system as `S390X` to avoid confusion in the future, as searching for `S390` on it's own brings you to the earlier iteration that, at first glance support was discontinued for in 1998.